File: 백준/Bronze/9366. 삼각형 분류/삼각형 분류.cc
#include <iostream>
#include <vector>
using namespace std;
int main(void) {
    int n, maxn, total;
    cin >> n;
    vector<int> arr(3);
    for (int i = 1; i < n + 1; i++) {
        maxn = 0, total = 0;
        for (int j = 0; j < 3; j++) {
            cin >> arr[j];
            if (maxn < arr[j]) maxn = arr[j];
            total += arr[j];
        }
        if (arr[0] == arr[1] && arr[1] == arr[2] && arr[2] == arr[0])
            cout << "Case #" << i << ": equilateral\n";
        else if (maxn < total - maxn)
            cout << "Case #" << i << (arr[0] == arr[1] || arr[1] == arr[2] || arr[0] == arr[2] ? ": isosceles" : ": scalene") << "\n";
        else
            cout << "Case #" << i << ": invalid!\n";
    }
    return 0;
}

File: 백준/Bronze/9366. 삼각형 분류/삼각형 분류.py
import sys
input=lambda:sys.stdin.readline().rstrip()
for i in range(1,int(input())+1):
    arr=[*map(int,input().split())]
    maxn=max(arr)
    if arr[0]==arr[1]==arr[2]: print(f"Case #{i}: equilateral")
    elif maxn<sum(arr)-maxn: print(f"Case #{i}: {'isosceles' if arr[0]==arr[1] or arr[1]==arr[2] or arr[0]==arr[2] else 'scalene'}")
    else: print(f"Case #{i}: invalid!")